atd-vzv/src/views/summary/CrashesByMode.js

```diff
@@ -222,7 +224,9 @@ const CrashesByMode = () => {
     <Container className="m-0 p-0">
       <Row>
         <Col>
-          <h2 className="text-left, font-weight-bold">By Travel Mode</h2>
+          <h2 className="text-left, font-weight-bold">By Travel Mode{" "}
```

Comment:
Could you removed the comma next to this `text-left` class since you are in here? That snuck its way in there somewhere in the past.

Also, no change needed, but I noticed we use a space character around the app to create space between elements. Probably something we could have reached to CSS styles to achieve!

Reply:
thanks for pointing that out, Mike. i would also rather see spacing addressed here with a padding class rather than `" "`, but i guess we should tackle that separately so that we can fix it everywhere at once?

Reply:
thanks, y'all! @mddilley I dropped the comma, thanks for noticing that! and +1 to fixing the spacing using css throughout the app. going to go ahead and merge for the next release !